app/src/main/java/com/example/triptracker/view/profile/UserProfileSettings.kt

```diff
@@ -205,7 +206,7 @@ fun UserProfileSettings(
                           },
                           colors =
                               ButtonDefaults.filledTonalButtonColors(
-                                  containerColor = md_theme_grey,
+                                  containerColor = md_theme_warning_orange,
```

Comment:
What do you think of changing the color ? I think that this orange is too close to the theme one.